### PR TITLE
Add `Project Prism 2024` tournament article

### PR DESCRIPTION
**Pull request by Walavouchey**
[web preview](https://osu3.roanh.dev/wiki/en/Tournaments/Project_Prism/2024)

### CODE REVIEW

File: wiki/Tournaments/Project_Prism/2024/en.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+Detailed information can be found on the [website](https://prism.osumalaysia.org/raid-battle/info).
+
+## Ruleset
+
+### Registrations
+
+1. Registrations must be submitted through the [website](https://prism.osumalaysia.org/registration).
+2. All registered players will undergo an initial screening process by the osu! staff and any players who do not pass the screening will not be eligible to participate in the tournament. Results of the screening will be announced in the Discord server.
+3. Players **must** join the [Discord server](https://discord.com/invite/sJK3rMdfwC) to be eligible for the tournament.
```
Comment (iSlodinxOsu) [inline wiki/Tournaments/Project_Prism/2024/en.md:79]:
```suggestion
3. Players **must** join the [Discord server](https://discord.com/invite/osutaikomy) to be eligible for the tournament.
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
perhaps it should be with vanity URL instead?

File: wiki/Tournaments/Project_Prism/2024/en.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+1. Players are required to be present at the times of their match. A reminder in the form of a Discord ping will be given to the players 15 minutes before every match.
+2. Invites to the room will then be sent out 10 minutes before each match.
+3. Failure to arrive on time will result in the following penalties:
+   - 5 minutes: Player warmup will be skipped
+   - 10 minutes: Player map ban goes to opponent
+   - 15 minutes: Player loses by default
+4. Warmups must be sent to the referees at least 30 minutes before the start of the map. The warmup must be less than 3 minutes in length and not contain inappropriate content. The referee reserves the right to reject maps that they deem to be inappropriate.
+5. At the start of each match, players will be asked to `!roll` to determine who picks or bans first. The player with the higher roll will get to pick first and ban second.
+6. Banned maps and tiebreakers are not allowed to be picked throughout the match.
+7. Each player will be given a maximum of 2 minutes to select their picks. If a pick has not be decided before the countdown timer ends, the map will be randomised with `!roll`.
```
Comment (0x84f) [inline wiki/Tournaments/Project_Prism/2024/en.md:128]:
```suggestion
7. Each player will be given a maximum of 2 minutes to select their picks. If a pick has not been decided before the countdown timer ends, the map will be randomised with `!roll`.
```

File: wiki/Tournaments/Project_Prism/2024/en.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+8. After a map is decided, players will then have a maximum of 2 minutes to ready up before the referee forces a start.
+9. Players should ensure that they have all the maps downloaded before the start of their match. In the event that a player is missing a map, they will automatically lose the point if they are unable to download it within the 2 minutes given.
+10. Players may ask for a "tactical timeout" during a match, which will add an extra 2 minutes to the timer. This can only be used **once** per match.
+11. A rematch of the same map will be given if a player disconnects within the first 30 seconds of the map. Any subsequent disconnections on the same map will be treated as a point loss.
+12. Lag is not a valid reason to restart a map.
+13. In the event of a tie in scores, neither player will gain points and the map will be played again.
+14. In the unlikely event that both players get a tied score twice in a row on the same map, the map will be nullified and another map needs to be picked by the other player.
```
Comment (0x84f) [inline wiki/Tournaments/Project_Prism/2024/en.md:135]:
```suggestion
14. In the unlikely event that both players get a tied score twice in a row on the same map, the map will be nullified, and another map needs to be picked by the other player.
```

**Pull request merged**